esmvaltool/cmorizers/obs/cmor_config/Duveiller2018.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Common global attributes for Cmorizer output
+attributes:
+  dataset_id: Duveiller2018 
+  version: 'v2018' 
+  tier: 2
+  modeling_realm: clim
+  project_id: custom
+  source: 'https://ndownloader.figshare.com/files/9969496'
+  reference: 'duveiller2018'
+  comment: ''
+  climatology_start: '2008-01-01T00:00:00Z'
+  climatology_end: '2012-12-31T23:59:59Z'
+
+# Variables to cmorize
+variables:
+  albDiff:
+    mip: Amon
+    # Match CMOR variables with input file one
+    raw: Delta_albedo
+    # input file name
+    file: albedo_IGBPgen.nc
+
+# This script has one parameter
+parameters:
+  iTr: 13 # The vegetation cover transition code (variable name iTr on the original file) to extract, see data source for explanation.
```

Comment:
Wouldn't it make more sense to CMORize all the codes and put them in the variable name somehow? Because if you use `albDiff` in a recipe, it's not clear which iTr you mean and it will not be possible to have more than one.

Reply:
> Wouldn't it make more sense to CMORize all the codes and put them in the variable name somehow? 
Yes, I agree, will go for that. @mattiarighi should I simply add them to the custom CMOR table? Since these are so dataset specific, should we maybe specify somewhere in the CMOR_albDiffiTr12.dat files that these definitions are specific to this dataset? Or maybe even have a separate folder for them? Let me know if you have ideas how to organize this. I think it would be good to keep this folder organized such that people can actually see if a certain custom CMOR entry is there already.

Reply:
@mattiarighi  Any input? If not, I just move forward.

Reply:
For now, I just added one specific vegetation cover transition code (13).